src/ORM/Association/BelongsToMany.php

```diff
@@ -118,9 +118,9 @@ class BelongsToMany extends Association
     /**
      * The table instance for the junction relation.
      *
-     * @var \Cake\ORM\Table|string
+     * @var \Cake\ORM\Table|string|null
      */
-    protected Table|string $_through;
+    protected Table|string|null $_through = null;
```

Comment:
isn't that a BC break? 🤔

Reply:
It is on an internal protected prop.
if someone extended this method, it is unlikely they also overwrote this property.

Currently, there is a hard PHP error. So the bugfix here might be warranted for now.

Reply:
I don't users can or should extend the existing associations.